lib/taskana-core/src/test/java/acceptance/task/CompleteTaskAccTest.java

```diff
@@ -45,10 +87,11 @@ void testCompleteTask() throws Exception {
 
   @WithAccessId(user = "admin")
   @Test
-  void should_completeClaimedTaskByAnotherUser_When_UserIsAdmin() throws Exception {
-    assertThat(taskService.getTask("TKI:000000000000000000000000000000000029").getState())
-        .isEqualTo(TaskState.CLAIMED);
-    Task completedTask = taskService.completeTask("TKI:000000000000000000000000000000000029");
+  void should_CompleteClaimedTaskByAnotherUser_When_UserIsAdmin() throws Exception {
+    Task task = createTaskClaimedByUser();
+
+    Task completedTask = taskService.completeTask(task.getId());
+
     assertThat(completedTask).isNotNull();
```

Comment:
Same thing as above: 

> I know you didn't modify this, but maybe you're willing to make an adjustment here ^^
Instead of asserting modified is not created I'd personally assert that completed = modify != null != created.